ddtrace/tracer/textmap_test.go

```diff
@@ -1609,3 +1610,210 @@ func TestNonePropagator(t *testing.T) {
 func assertTraceTags(t *testing.T, expected, actual string) {
 	assert.ElementsMatch(t, strings.Split(expected, ","), strings.Split(actual, ","))
 }
+
+func BenchmarkInjectDatadog(b *testing.B) {
+	b.Setenv(headerPropagationStyleInject, "datadog")
+
+	tracer := newTracer()
+	defer tracer.Stop()
+
+	root := tracer.StartSpan("test")
+	defer root.Finish()
+
+	for i := 0; i < 100; i++ {
+		setPropagatingTag(root.Context().(*spanContext), fmt.Sprintf("%d", i), fmt.Sprintf("%d", i))
+	}
+
+	dst := map[string]string{}
+
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		tracer.Inject(root.Context(), TextMapCarrier(dst))
+	}
+}
+
+func BenchmarkInjectW3C(b *testing.B) {
+	b.Setenv(headerPropagationStyleInject, "tracecontext")
+
+	tracer := newTracer()
+	defer tracer.Stop()
+
+	root := tracer.StartSpan("test")
+	defer root.Finish()
+
+	ctx := root.Context().(*spanContext)
+
+	traceID := "4bf92f3577b34da6a3ce929d0e0e4736"
+	oldTraceState := "othervendor=t61rcWkgMzE,dd=s:2;o:rum;t.dm:-4;t.usr.id:baz64~~"
+
+	setPropagatingTag(ctx, w3cTraceIDTag, traceID)
+	setPropagatingTag(ctx, tracestateHeader, oldTraceState)
+
```

Comment:
The variable is only used once, no need to declare
```suggestion
	setPropagatingTag(ctx, w3cTraceIDTag,  "4bf92f3577b34da6a3ce929d0e0e4736")
	setPropagatingTag(ctx, tracestateHeader,"othervendor=t61rcWkgMzE,dd=s:2;o:rum;t.dm:-4;t.usr.id:baz64~~"
)

```